basic/http/collect-user.test.js

```diff
@@ -1,13 +1,25 @@
 const request = require('supertest');
-const app = require('./collect-user');
+const { app, hashPassword,argon2 } = require('./collect-user');
```

Comment:
You can import argon2 here like this:
```
const request = require('supertest');
const argon2 = require('argon2');

const { hashPassword, app } = require('./collect-user');
```

We should prevent circular dependency

Reply:
RESOLVED 💯